common/src/main/java/com/wynntils/core/features/UserFeature.java

```diff
@@ -33,13 +33,20 @@ public final void updateConfigOption(ConfigHolder configHolder) {
     /** Updates the feature's enabled/disabled state to match the user's setting, if necessary */
     public final void tryUserToggle() {
```

Comment:
Why do we even have this function? It just seems so messy, all this enable/disable stuff.

Reply:
Dealing with enabling is best left for a separate pr, and I slightly agree with your statement.